### DISCUSSION TITLE
Fejlesztői dokumentáció

### DISCUSSION THREAD

**Discussion by wlln22** [nav-gov-hu/eRECEIPT#36] [Kérdések és válaszok]
Amennyiben a Fejlesztői Dokumentáció tartalmának változása (pl. NAV-I interfész vagy szolgáltatás módosulása) miatt szükséges az epg szoftverváltoztatása, akkor is díjköteles a forgalmazási engedély módosítása?

A fejlesztői dokumentációban nem szerepel leírás e-pénztárgépről kiolvasott bizonylatok értelmezését segítő programról. Várható-e, hogy bekerül a Fejlesztői Dokumentációba egy funkcionalitás-lista erre vonatkozóan?

**Comment by BatariZ** [comment]
Az időközben megjelent rendelet (A nemzetgazdasági miniszter 8/2025. (III. 31.) NGM rendelete az e-pénztárgépek forgalmazásáról, üzemeltetéséről, valamint az e-pénztárgépek és az e-nyugta kiállításának követelményeiről), illetve az aktualizált Fejlesztői dokumentáció tartalmazza a válaszokat a feltett kérdésekre.
